deepnote_toolkit/sql/jinjasql_utils.py

```
@@ -40,9 +40,14 @@ def _get_variable_value(variable_name):
     return getattr(__main__, variable_name)
 
 
-def _escape_jinja_template(template):
+def _escape_jinja_template(template, param_style: str = "pyformat"):
```

Ruff (0.14.4)

Missing return type annotation for private function `_escape_jinja_template`

(ANN202)